src/plugins/embeddable/public/lib/embeddables/error_embeddable.scss

```diff
@@ -1,10 +1,11 @@
-
 .errorEmbeddableCompact__popover {
   height: 100%;
   max-width: 100%;
 }
 
 .errorEmbeddableCompact__button {
+  height: $euiButtonHeightSmall;
+
```

Comment:
@elastic/kibana-presentation team, this was done to be consistent with below `optionsList` styles.

https://github.com/elastic/kibana/blob/fabe07d94ed5a7fa90a7e5e14f8806607ddf543a/src/plugins/controls/public/options_list/components/options_list.scss#L9-L12